src/Elemental.js

```diff
@@ -1,3 +1,4 @@
+require('babel/register');
```

Comment:
this isn't safe for node libraries to use; it causes _all_ require statements in an app to go through babel's transformation process. that's why the entry point in `package.json` is pointed at the transpiled `lib` directory.